pages/article/ArticleHandler.inc.php

```diff
@@ -17,18 +17,30 @@
 import('classes.handler.Handler');
 
 class ArticleHandler extends Handler {
-	/** journal associated with the request **/
+	/** @var journal object: journal associated with the request **/
 	var $journal;
```

Comment:
This variable is not used much. Can all the other journal variables then be replaced with this one? -- it seems to be a little bit inconsistent...